packages/atst/src/lib/abi.ts

```diff
@@ -0,0 +1,133 @@
+export const abi = [
```

Comment:
Before release I will make it so we import this from artifacts.   I couldn't get typescript to work correctly while doing that though so I did this as a workaround.   At some point I will have wagmi cli generate some code for me and this problem will fix itself